src/style.css

```diff
@@ -0,0 +1,85 @@
+h1 {
+  text-align: center;
+  font-size: 50px;
+}
+
+.list-name {
+  width: 40%;
+}
+
+.name-head {
+  display: flex;
+  justify-content: space-between;
+  width: 35%;
+  margin-left: 27rem;
+}
+
+.name-head button {
+  padding: 3px 15px;
+  height: 50px;
+  border-radius: 4px;
+  border: none;
+}
+
+.name-head h3 {
+  font-size: 30px;
+}
+
+.btn-div {
+  align-self: center;
+}
+
+.name-list li {
+  list-style: none;
+  width: 95%;
+  font-size: 25px;
+  display: flex;
+  justify-content: space-between;
+  padding: 15px;
+  margin-bottom: 1rem;
+  border-radius: 4px;
+  background-color: rgb(245, 245, 245);
+}
+
+.name-list {
+  border: 1px solid #7d53de;
+  padding: 20px;
+  margin: 10px;
+}
+
+.box {
+  width: auto;
+  max-width: 300px;
+  margin: 20px auto;
+  display: block;
+  padding: 8px;
+  border: 1px solid #ddd;
+  border-radius: 4px;
+  font-size: 16px;
+  color: #444;
+}
+
+.main-container {
+  display: flex;
+  justify-content: center;
+  gap: 3rem;
+}
+
+#formm.sub {
+  width: 10%;
+  margin-right: 320px;
+  cursor: pointer;
+}
+
+#formm div {
+  width: 10%;
+  position: relative;
+  left: 57%;
+}
+
+#formm div .sub:hover {
```

Comment:
- [ ] Let's not use id as a selector in CSS. IDs shouldn't be used in selectors because these rules are too tightly coupled with the HTML and have no possibility of reuse.